package.json

```diff
@@ -39,7 +39,10 @@
     "capture"
   ],
   "dependencies": {
-    "pify": "^2.3.0"
+    "ffi": "^2.2.0",
+    "pify": "^2.3.0",
+    "ref": "^1.3.4",
+    "ref-wchar": "^1.0.2"
```

Comment:
Can you make the new dependencies `optionalDependencies`, so it doesn't fail to install on macOS/Linux if something is missing on the user's system to compile.